fullstack/js/price-calculator/app/managers/cart-manager.js

```diff
@@ -0,0 +1,27 @@
+import CartItem from '../models/cart-item'
+
+export default class CartManager {
+  data = []
+  basePriceManager
+
+  constructor(basePriceManager, jsonFile) {
+    this.data = this.readJson(jsonFile)
+    this.basePriceManager = basePriceManager
+  }
+
+  readJson(jsonFile) {
+    let data = []
+    const rawData = require(jsonFile)
+
+    for (const rawItem in rawData) {
+      const cartItem = new CartItem(rawItem)
```

Comment:
Should the for-loop use `of`? Also, the `CartItem` constructor wouldn't work here I think

Reply:
Yes, I'll have to provide arguments to `CartItem` 👍